src/Swift.Bindings/src/Emitter/StringEmitter/Handler/MethodHandler.cs

```diff
@@ -159,7 +159,7 @@ public record Parameter(string Type, string Name, string modifier = "")
     /// <param name="Parameters"></param>
     public record Signature(string ReturnType, IReadOnlyList<Parameter> Parameters)
     {
-        public bool ContainsPlaceholder => Parameters.Any(p => p.Type == "AnyType") || ReturnType == "AnyType";
+        public bool ContainsPlaceholder => Parameters.Any(p => p.Type.Contains("AnyType") || ReturnType.Contains("AnyType"));
         public string ParametersString() => string.Join(", ", Parameters.Select(p => p.SignatureString()));
```

Comment:
This check would return true even if the return type (for example) is `MyAnyType` - which would be wrong. If the type names are qualified, then we should use the namespace qualified name for comparison. If this is about generic instantiation, then we need to have a different mechanism do detect generics with AnyType in them.

Reply:
This is both about generic instatiation and non generic type handling. A consequence of us not yet implementing the transactional emission model. The current solution which we have when we are not able to reason about something is to place this `AnyType` type on the signature. If the signature contains this placeholder we know that we need to skip this method.

We could keep AnyType for now, make sure it is referenced with fully qualified name. For bound generics emit AnyType if any of the generic params is unknown. Moving forward - when implementing the transactional model - we should review whether we even need `AnyType`

cc @kotlarmilos

Reply:
It sounds like we're ready to iterate on the design :)

Reply:
Modified the logic to check the fully qualified named instead. This should avoid problems outlined by @vitek-karas. Lets discuss the future of `AnyType` separately.